.changeset/@graphql-yoga_apollo-managed-federation-3532-dependencies.md

```diff
@@ -0,0 +1,6 @@
+---
+"@graphql-yoga/apollo-managed-federation": patch
```

Comment:
Seems like `federation` pkg has major version upgrade here, it does not affect other things? should we consider to bump major?

Also, I think we should have clear documentation of the changes and how they affect the current package in Yoga (even if it's just an external link)

Reply:
See https://github.com/dotansimha/graphql-yoga/pull/3532#discussion_r1878195178